snuba/datasets/storages/transactions.py

```diff
@@ -209,6 +210,7 @@ def transactions_migrations(
             {"start_ts", "finish_ts"},
             BEGINNING_OF_TIME,
         ),
+        TransactionColumnProcessor(),
```

Comment:
Do you plan to send a PR to add this processor to all the others datasets that extend TimeSeriesDataset ?

Reply:
No I'll add it here.